vendor/phantomjs/render.js

```diff
@@ -55,7 +55,8 @@
 
         var rootScope = body.injector().get('$rootScope');
         if (!rootScope) {return false;}
-        return rootScope.panelsRendered;
+        var panelsToLoad = window.angular.element('div.panel').length;
```

Comment:
It must be like this:
var panelsToLoad = window.angular.element('div.panel').length - window.angular.element('div.panel-drop-zone').length;
                        return rootScope.performance.panelsRendered >= panelsToLoad;
Because some 'div.panel' elements not real panels.